lib/matplotlib/pyplot.py

```diff
@@ -2003,253 +2003,6 @@ def get_plot_commands():
            and inspect.getmodule(obj) is this_module)
 
 
-def colormaps():
-    """
```

Comment:
Should this docstring be copied (with slight editing) into `ColormapRegistry.__doc__`, e.g. into a Notes section (so that it is still available e.g. under ipython with `plt.colormaps?`)

Reply:
I don't think so. The old docstring is a bit tutorial plus a long description of all available colormaps. This is essentially all covered in the [colormap reference](https://matplotlib.org/stable/gallery/color/colormap_reference.html#sphx-glr-gallery-color-colormap-reference-py) and the [colormap tutorials](https://matplotlib.org/stable/tutorials/index.html#colors). The docstring of `ColormapRegistry` should essentially be limited to the API.

Reply:
I guess the question is, do we want to have a description of all colormaps available "offline" from just standard docstrings.  But perhaps that would be better served by having some colormap metadata anyways...  so that discussion can happen later.